content/manager/bootstrapping.md

```diff
@@ -165,6 +165,153 @@ Services:
+    | new_broker_username             |           | ✓           | New username for the message broker.                                                                                                                                                                                                                                     |
+    | new_broker_password             |           | ✓           | New password for the message broker.                                                                                                                                                                                                                                     |
+
+    {{% gsNote %}}
+    It is possible to use the contents of an inputs file (if formatted as JSON) directly by pasting them into the "raw" field.
+    {{% /gsNote %}}
+
+ 4. Once the new deployment is ready, execute the `install` workflow:
+    ![CloudifySettings deployment: install workflow]({{< img "manager/image-deploy-run-install.png" >}})
+
+    Once you have started the `install` workflow it will take a few minutes to run.
+    If you have chosen a security-enabled build the Web UI will become unresponsive as several services need to be restarted. Depending on your web browser it may be necessary to refresh the whole page. You will also need to accept a new self-signed certificate as the certificate is regenerated as part of the `CloudifySettings` blueprint.
+
+
+## Command Line
+
+ 1. Switch to the new manager:
+
+    ```bash
+    $ export CLOUDIFY_USERNAME=cloudify
+    $ export CLOUDIFY_PASSWORD=cloudify
+    $ export CLOUDIFY_SSL_TRUST_ALL=true
+    $ cfy use --port 443 --management-ip ${CLOUDIFY_HOST}
+    ```
+
+    {{% gsNote title="TLS" %}}
+    If you chose one of the `-insecure` master images then use port 80 instead (in this case the `export` lines above are not required): `$ cfy use --port 80 --management-ip ${CLOUDIFY_HOST}`.
+
+    {{% /gsNote %}}
+
+ 2. Create an input file for the `CloudifySettings` blueprint
+    ```bash
+    $ echo >install.yaml <<EOF
+    {
+      "aws_access_key": "{your access key}",
+      "aws_secret_key": "{your secret key}",
+      "agents_security_group_name": "{your desired group}",
+      "user_ssh_key": "{your ssh key}",
+      "agents_user": "centos",
+      "agents_keypair_name": "{your desired keypair name}",
+      "new_manager_username": "cloudify",
+      "new_broker_password": "a bad password",
+      "broker_names_and_ips": "localhost",
+      "new_broker_username": "cloudify",
+      "manager_names_and_ips": "${CLOUDIFY_HOST}",
+      "new_manager_password": "a really bad password"
+    }
+    EOF
+    ```
+ 3. Create the deployment
+    ```bash
+    $ cfy deployments create --deployment-id install --blueprint-id CloudifySettings --inputs install.yaml
```

Comment:
deployment-id should be in the same 'variable' format, or default to CloudifySettings or something like that.